### PR TITLE
Add param for rotation of tool changer

### PR DESCRIPTION
**Pull request by AdamPettinger**
Easy, but breaks existing URDFs that use this

### CODE REVIEW

File: robotiq_85_description/urdf/tool_changer.urdf.xacro [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
-  <xacro:macro name="tool_changer" params="prefix connected_to">
+  <xacro:macro name="tool_changer" params="prefix connected_to rotation">
```
Comment (schornakj) [inline robotiq_85_description/urdf/tool_changer.urdf.xacro]: If you use the [xacro syntax for parameter defaults and forwarding](https://github.com/ros/xacro/pull/105), do you think you could make it so this modification doesn't break existing robot descriptions that use this macro?

Here's an [example macro definition](https://github.com/ros-controls/ros2_control_demos/blob/ead1af151d92b104774e9061684cf23b52f67507/ros2_control_demo_description/rrbot_description/ros2_control/rrbot_system_position_only.ros2_control.xacro#L4) and an [example usage](https://github.com/ros-controls/ros2_control_demos/blob/ead1af151d92b104774e9061684cf23b52f67507/ros2_control_demo_description/rrbot_description/urdf/rrbot_system_position_only.urdf.xacro#L40-L44).

Comment (AdamPettinger) [reply robotiq_85_description/urdf/tool_changer.urdf.xacro]: Yes! That worked great. Tested with the picknik UR5 URDF. Did with and without the new arg, and no complaints while loading the tool changer Xacro (and could see the default rotation when it was excluded)

Comment (schornakj) [reply robotiq_85_description/urdf/tool_changer.urdf.xacro]: Awesome!

File: robotiq_85_description/urdf/tool_changer.urdf.xacro [hunk 2]
```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <robot xmlns:xacro="http://ros.org/wiki/xacro">
-  <xacro:macro name="tool_changer" params="prefix connected_to">
+  <xacro:macro name="tool_changer" params="prefix connected_to rotation:=^|0">
```
Comment (schornakj) [inline robotiq_85_description/urdf/tool_changer.urdf.xacro]: Should the default rotation value be `pi/2.0` to match the original yaw value in the joint?

Comment (AdamPettinger) [reply robotiq_85_description/urdf/tool_changer.urdf.xacro]: Yea, good catch. Will update

**Comment by schornakj** [comment]
You should also do a rebase to get rid of the commits from the collision mesh PR and just have this depend on the main branch.

I think this would be like below (assuming your remote for the main repo is named "upstream"):
```
git rebase --onto upstream/foxy bf6980194cd681a13f6222c454b12e9c446dfae5 feature/tool_changer_rotation
```

**Review by schornakj — APPROVED** [review]

**Pull request merged**